### DISCUSSION TITLE
Pinned (Offline) Tabs?

### DISCUSSION THREAD

**Discussion by lurkusignoramus** [Chatterino/chatterino2#5238] [Ideas]
Whether it be for tabs like /mentions or /whispers, your own channel, or a community that chats offline regularly, it would be nice to be able to pin tabs so that they stay visible when offline while using the "only live tabs" setting. Hope that makes sense.

**Comment by Felanbird** [comment]
Duplicate of #5027

basically not possible right now unless some overhaul is done - workaround is to add a stream that's live 24/7 and make it really tiny
![image](https://github.com/Chatterino/chatterino2/assets/41973452/f77acc05-8a47-4669-b998-d780d09c04c8)
